wiki/Installation/en.md

```diff
@@ -0,0 +1,31 @@
+# Installation
+
+To begin playing on Titanic! you can head over to the [downloads page](https://osu.lekuru.xyz/download/) and pick any client from as old as 2007 up to 2014.
+
+![](https://i.ibb.co/WGqryjw/Screenshot-from-2024-05-27-17-53-40.png)
+
+After you install it Smart Screen may pop up when you open it, since the game is unsigned. (a certificate for that would cost 200$)
+
+After it finishes installing, all you have to do is extract the folder and double-click `osu!.exe`.
```

Comment:
```suggestion
After it finishes installing, all you have to do is extract the folder and double-click `osu!.exe`.

There is a chance that Windows Defender may block it, since the game is unsigned (a certificate for that would cost 200$).
```